src/main.ts

```diff
@@ -40,14 +40,14 @@ const EXTRA_CLI_ARGS = core.getInput('argocd-extra-cli-args');
 
 const octokit = github.getOctokit(githubToken);
 
-async function execCommand(command: string, failingExitCode = 1): Promise<ExecResult> {
+async function execCommand(command: string, options: ExecOptions = {}): Promise<ExecResult> {
```

Comment:
Change looks good, though unused?

Reply:
Yeah, turned out not to be needed, but this change seemed useful enough to keep.